src/ServiceControl.Audit.Persistence.RavenDb/CustomChecks/CheckMinimumStorageRequiredForAuditIngestion.cs

```diff
@@ -24,7 +24,13 @@ public override Task<CheckResult> PerformCheck()
                 return successResult;
             }
 
-            var dataPathRoot = Path.GetPathRoot(settings.PersisterSpecificSettings[RavenBootstrapper.DatabasePathKey]);
+            if (!settings.PersisterSpecificSettings.TryGetValue(RavenBootstrapper.DatabasePathKey, out var databasePath))
```

Comment:
@cquirosj this change is unrelated so I'll try to pull this out as a separate PR before we merge this